### DISCUSSION TITLE
Remove the Scorchling from the CC rewards catalogue.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#5843] [Suggestions]
This is a pet as a part of the Midsummer Fire Festival event as a drop from Ahune. Seeing as this event is being played out on CC (and may once again next year if progression doesn't speed past), this pet should not be available outside of the event.

**Comment by Amrx2** [comment]
Hi Bench, valid point and has been removed from the shop.
